examples/sql/driver/json/main.go

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+This example demonstrates how to use the Hazelcast database/sql driver.
```

Comment:
Would it make sense to mention here that this example works only with Hazelcast 5.1 and up?

Reply:
Sure, added at https://github.com/hazelcast/hazelcast-go-client/pull/726/commits/972309f6841a6c712338f131d67636935ede4025